Sources/SwiftGRPC/Runtime/ServiceClient.swift

```diff
@@ -60,7 +60,25 @@ open class ServiceClientBase: ServiceClient {
     self.channel = channel
     metadata = Metadata()
   }
-  
+
+  /// Create a client with Google credentials.
+  /// - Parameter googleApi: the name of the Google API service (e.g. "cloudkms" in "cloudkms.googleapis.com")
+  /// - Parameter arguments: list of channel configuration options
+  ///
+  /// Note: cgRPC's `grpc_google_default_credentials_create` doesn't accept a root pem argument.
+  /// To override: `export GRPC_DEFAULT_SSL_ROOTS_FILE_PATH=/path/to/your/root/cert.pem`
+  required public init(googleApi: String, arguments: [Channel.Argument] = []) {
```

Comment:
Nit: `googeAPI`?

Also, are _all_ Google APIs namespaced under the `.googleapis.com` domain?

Reply:
Grepping through https://github.com/googleapis/googleapis, it appears that the answer is indeed yes: `ag '^name:' -G '.*.yaml'`